Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Mentioning that -p with the tcptraceroute wrapper specifies the source port,
 not the destination port (Closes: #448200).

diff -Naurp traceroute.orig/traceroute/traceroute.8 traceroute/traceroute/traceroute.8
--- traceroute.orig/traceroute/traceroute.8	2012-12-10 11:49:37.056140846 +0100
+++ traceroute/traceroute/traceroute.8	2012-12-10 11:49:46.172143321 +0100
@@ -198,7 +198,8 @@ For ICMP tracing, specifies the initial
 by each probe too).
 .br
 For TCP and others specifies just the (constant) destination
-port to connect.
+port to connect. When using the tcptraceroute wrapper,
+\-p specifies the source port.
 .TP
 .BI \-t " tos
 For IPv4, set the Type of Service (TOS) and Precedence value. Useful values
